docs/CAVE.md

```diff
@@ -0,0 +1,158 @@
+# Unity Setup for CAVE Deployment
+
+## Writing3D Notes
+
+- Root object needs to move to be the first in the hierarchy
+- Add MVR Interactable script whenever I add a LinkManager
```

Comment:
Is MVR MiddleVR? Maybe define that somewhere?

Reply:
Yeah, most of their scripts started with `MVR`. I can add a note at the top of the doc